tests/splineContourSegmentationTools.spec.ts

```diff
@@ -5,104 +5,6 @@ import {
   screenShotPaths,
 } from './utils/index';
 
-async function updateSplineStyleInputs({ page, splineStyle }) {
```

Comment:
You can ignore the changes to this file because I just moved the test functions to the top.